roles/network_plugin/calico/meta/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+dependencies:
+  - role: network_plugin/cni
+    when:
+      - enable_cni
+      - skip_cni is not defined
```

Comment:
I am not sure why we need to have 2 options `skip_cni` and `enable_cni`.
Doesn’t `enable_cni: false` mean skipping cni installation?

Reply:
@oomichi  When executing calico's `check` or `reset` task, the `dependencies` in meta will be executed first, which is not expected, so I skip by defining the `skip_cni` variable, and `enable_cni` may be redundant, I plan to delete it.

Reply:
@oomichi  😊 Thanks, I have updated.